.circleci/config.yml

```diff
@@ -7,12 +7,14 @@ jobs:
       - image: automattic/wp-e2e-tests:0.0.8
         environment:
                 NODE_ENV: test
+                NODE_CONFIG: {}
     steps:
       - checkout
       - restore_cache:
           key: << checksum "package.json" >>
       - run: source $HOME/.nvm/nvm.sh && npm install
       - run: source $HOME/.nvm/nvm.sh && npm run lint
+      - run: source $HOME/.nvm/nvm.sh && openssl enc -aes-256-cbc -d -k $CONFIG_KEY -in ./config/encrypted.enc -out ./config/local-test.json
```

Comment:
I think a better way would be `openssl enc -aes-256-cbc -d -pass env:CONFIG_KEY -in ./config/encrypted.enc -out ./config/local-test.json`, since that won't risk accidentally dumping the config key to the screen (and CircleCI UI) if the command fails.